state/execution.go

```diff
@@ -112,14 +112,40 @@ func (blockExec *BlockExecutor) CreateProposalBlock(
 	// Tx -> Txs, Message
 	// https://github.com/lazyledger/lazyledger-core/issues/77
 	txs := blockExec.mempool.ReapMaxBytesMaxGas(maxDataBytes, maxGas)
+	l := len(txs)
+	bzs := make([][]byte, l)
+	for i := 0; i < l; i++ {
+		bzs[i] = txs[i]
+	}
```

Comment:
A bit annoying that we have to cast between `types.Txs` and  `[][]byte` here 🤔

Reply:
yea, there is an issue to decide that discusses what to do here but for now its all that can be done. I can abstract this away but didnt see the need.